theme/js/script.js

```diff
@@ -150,4 +152,23 @@ function validateForm() {
             requiredField.siblings('label').addClass('invalid');
         }
     });
-}
\ No newline at end of file
+}
+
+function getFormData() {
+    $.getUrlParam = function(name){
+        // Variable results is set to equal to the result of the search using RegExp.
+        // RegExp defines the characters which can be before the word, the word, and characters which can appear after the word.
+        // RegExp is the search term used by the .exec function which looks for the result in the url.
+        var results = new RegExp('[\?&]' + name + '=([^&#]*)').exec(window.location.href);
+        if (results==null) {
```

Comment:
Add a single space on either side of '=='

Reply:
done